docs/core/diagnostics/dotnet-stack.md

```diff
@@ -5,7 +5,7 @@ ms.date: 04/21/2021
 ---
 # Inspect managed stack traces (dotnet-stack)
 
-**This article applies to:** ✔️ .NET Core 3.0 and later versions
+**This article applies to:** ✔️ dotnet-stack version 7 and later versions
```

Comment:
I'd guess this info accurate for when dotnet-stack was initially released: https://github.com/dotnet/diagnostics/releases/tag/v5.0.221401
```suggestion
**This article applies to:** ✔️ dotnet-stack 5.0.221401 and later versions
```

Reply:
for dotnet-stack 5.0.221401 is the first time it was released

Reply:
What were the first versions released of the other tools?

Reply:
https://github.com/dotnet/diagnostics/releases/tag/3.0.47001